### PR TITLE
13765 BC Registry UI: Include the support contact information

### PR DESCRIPTION
**Pull request by lewischenstudio**
*Issue #:*
https://github.com/bcgov/entity/issues/13765

*Description of changes:*
- added support contact information in the footer
- UI design: https://preview.uxpin.com/e68d77b708da0ebbc185073ee5991ef2a90660be#/pages/159212743
- updated README instruction

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: README.md
Comment (severinbeauvais) [inline README.md:10]: Yes, good idea, but move the new instructions below the npm install line (since it is currently between the nvm and npm lines and isn't related to those).

**Comment by lewischenstudio** [comment]
This is the screenshot of the finished work in dev:

Screenshot 2022-11-01 112941

File: components/ContactInfoBcros.vue
Comment (severinbeauvais) [inline components/ContactInfoBcros.vue:59]: Please check with UI team whether "time" should be capitalized as in the design.

Also on line 43 above.

File: README.md
Comment (lambert-alex) [reply README.md:10]: I also like this addition.

File: components/ContactInfoBcros.vue
Comment (lewischenstudio) [reply components/ContactInfoBcros.vue:59]: I believe you are right. It is capitalized as "Time" in the design.

**Review by ChrisGabel23 — APPROVED** [review]

File: README.md
Comment (severinbeauvais) [inline README.md:8]: Typo at end of line.

The text of line 10 looks good to me but you don't need to prepend the ellipsis since it's not part of the previous topic.

File: components/ContactInfoBcros.vue
Comment (severinbeauvais) [reply components/ContactInfoBcros.vue]: Did you check with them? We don't capitalize "time" in other UIs. The design might be wrong.

File: README.md
Comment (lewischenstudio) [reply README.md:8]: ok 👌

File: components/ContactInfoBcros.vue
Comment (lewischenstudio) [reply components/ContactInfoBcros.vue]: Actually we do have "Time" in the [auth-web UI](https://dev.bcregistry.ca/business/auth/home/decide-business). I'll check with Yui.

**Review by lambert-alex — APPROVED** [review]
LGTM just resolve the other reviewers comments before merging

File: README.md
Comment (severinbeauvais) [reply README.md:8]: You still have that typo, ie:

image

Comment (lewischenstudio) [reply README.md:8]: You mean the backslash "\"? I wanted to make a new line after "below."

Comment (lewischenstudio) [reply README.md]: I removed the backslash. Looks like "double enter" can already create a new line.

Comment (severinbeauvais) [reply README.md]: You can view the file in GH to see it rendered, or add VS Code extension "Markdown Preview Enhanced" to see it.

File: components/ContactInfoBcros.vue
Comment (lewischenstudio) [reply components/ContactInfoBcros.vue:59]: I just received confirmation from Yui that we should use lowercase "time". It's now reverted to lowercase "time".

**Review by severinbeauvais — APPROVED** [review]
LGTM.

@doug-lovett , please have a look, run the GCP build, merge, deploy, etc. Thanks!

**Comment by lewischenstudio** [comment]
Please do not merge this PR yet. The design team is asking whether to use two "Hours of Operation" or to keep just one of them in case both of them are the same.

**Comment by lewischenstudio** [comment]
Just got a confirmation from the design team that only one "Hour of Operation" is required.

**Comment by severinbeauvais** [comment]
> Just got a confirmation from the design team that only one "Hour of Operation" is required.

OK. This code change looks good to me.

**Comment by ChrisGabel23** [comment]
> Just got a confirmation from the design team that only one "Hour of Operation" is required.

Yes, but which hour should we be in operation? ;) I'll show myself out.

**Pull request merged**
